{"name": "test_batch_blackList[args4]", "status": "passed", "steps": [{"name": "执行接口", "status": "passed", "start": 1715834315178, "stop": 1715834315485}, {"name": "响应体断言", "status": "passed", "start": 1715834315485, "stop": 1715834315487}], "attachments": [{"name": "log", "source": "e5b643f2-ef24-4f3a-aee8-6c8783afdc3b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'caseName': '【拉黑用户至黑名单接口】用例5：openid为无效汉字', 'method': 'post', 'url': '/cgi-bin/tags/members/batchblacklist', 'data': {'openid_list': '测试数据'}, 'excepted': {'statusCode': 200, 'errcode': 40003, 'errmsg': 'invalid openid'}}"}], "start": 1715834315178, "stop": 1715834315491, "uuid": "1b8e6ea7-480d-47ac-a8ff-6d9e13a48e74", "historyId": "bc0e68a6835f12a0cdc8fb94e5976a58", "testCaseId": "49e1c317b0bd0f105d2662a3d2e91abc", "fullName": "testcase.black_manage.test_batch_to_black.Test_userBlack#test_batch_blackList", "labels": [{"name": "feature", "value": "黑名单管理模块"}, {"name": "epic", "value": "微信项目"}, {"name": "story", "value": "拉黑用户至黑名单接口"}, {"name": "parentSuite", "value": "testcase.black_manage"}, {"name": "suite", "value": "test_batch_to_black"}, {"name": "subSuite", "value": "Test_userBlack"}, {"name": "host", "value": "DESKTOP-IG8PJM0"}, {"name": "thread", "value": "4692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.black_manage.test_batch_to_black"}]}